src/plugin/regions/region.js

```diff
@@ -708,7 +713,16 @@ export class Region {
             delta = this.start * -1;
         }
 
+        let activeDirection;
```

Comment:
not sure why you're redeclaring it here?

Reply:
Ok, you are right, I can simply update the `this.activeDirection` without `update`